### PR TITLE
Update create-retention-policies.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a work around when using more than 1000 mailboxes per inclusion/exclusion.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/3840

File: microsoft-365/compliance/create-retention-policies.md [hunk 1]
```diff
@@ -260,6 +260,8 @@ Only if you use the optional configuration to scope your retention settings to s
   - 1,000 users for Teams private chats
   - 100 sites (OneDrive or SharePoint)
 
+The workaround is be to use groups instead when exceeding 1000 mailboxes per inclusion/exclusion.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/create-retention-policies.md]:
```suggestion
The workaround is to use Microsoft 365 groups instead when exceeding 1000 mailboxes per inclusion/exclusion.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@cabailey Please merge this PR. Thanks.

**Comment by cabailey** [comment]
I'm closing this PR because this is not a valid workaround - the 1,000 mailboxes is for any type of mailbox, whether it's a user mailbox or group mailbox. I've got an update in progress that makes this clearer and emphasizes that if you hit the limitations, you can create multiple retention policies (with the tradeoff of additional admin overheads), and some concrete examples for valid workarounds.

**Comment by cabailey** [comment]
The update for this page is now live.

**Pull request closed without merging**